sympy/functions/elementary/exponential.py

```diff
@@ -540,6 +540,8 @@ def _eval_as_leading_term(self, x, logx=None, cdir=0):
             # AccumBounds(-oo, 0) or AccumBounds(-oo, oo).
             # Check out function: test_issue_18473() in test_exponential.py and
             # test_limits.py for more information.
+            if re(cdir) < S.Zero:
+                return 1/exp(arg0)
```

Comment:
Looks good , maybe we could make this `exp(-arg0)` and add few test cases like
```
>>> limit((tan(x)**2)**(2/x) , x, -oo)
AccumBounds(0, oo)
```
should be enough here , will merge this then !

Reply:
Okay sure. The `ci/circleci: Build Error` here seems irrelevant to the changes.

Reply:
I have made the changes, could go in after the tests run.